Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -132,9 +132,9 @@ The CSI-Addons Controller handles the requests from users to initiate an operati
 Users can deploy the controller by running the following commands:
 
```

Comment:
Instead of hardcoding version use latest release tag?

Reply:
> Instead of hardcoding version use latest release tag?

I am not sure having latest release tag is a good idea, 
since a tagged rook version will have a default older k8s-sidecar 
which should work alongside the closest release k8s-controller version.

Reply:
Yes, that's also true. its hard to find how many places we need to update when we have a new release :facepalm: